src/main/java/org/jitsi_modified/impl/neomedia/rtp/MediaSourceDesc.java

```diff
@@ -15,92 +15,90 @@
  */
 package org.jitsi_modified.impl.neomedia.rtp;
 
+import org.jitsi.nlj.*;
 import org.jitsi.nlj.rtp.*;
-import org.jitsi.nlj.rtp.codec.vp8.*;
 import org.jitsi.utils.*;
 
 import java.util.*;
 
 /**
- * Represents a collection of {@link RTPEncodingDesc}s that encode the same
+ * Represents a collection of {@link RtpLayerDesc}s that encode the same
  * media source. This specific implementation provides webrtc simulcast stream
```

Comment:
Can we define a 'media source' here?  I've already forgotten if we defined this as 'camera' or 'rtp stream'

Reply:
"All media encoded from a camera", yeah.  I'm trying to follow the RFC 7656 terminology.  I'll clarify.